requirements.txt

```diff
@@ -1,3 +1,6 @@
 pgi==0.0.11.1
 watchdog==0.10.3
 pyglet==1.2.4
+six==1.10.0
+mock==2.0.0
```

Comment:
Is `six` and `mock` really needed?

Reply:
I made this code 2.5 years ago. I have no idea.

Reply:
Both can be removed

Reply:
```suggestion
```